### PR TITLE
fix(823): [1] Add some release fields for SCHEMA_HOOK

### PR DESCRIPTION
**Pull request by kkokufud**
## Context
SD can execute release/tag trigger, but it can't provide release/tag info(ex. name).
SD user want to get release/tag info.

## Objective
Add some release fields to SCHEMA_HOOK.
Please see reference to confirm details.

## References
https://github.com/screwdriver-cd/screwdriver/issues/823#issuecomment-490014132

### CODE REVIEW

File: core/scm.js [hunk 1]
```diff
@@ -187,7 +187,24 @@ const SCHEMA_HOOK = Joi.object().keys({
...
-    username: Joi.reach(SCHEMA_USER, 'username')
+    username: Joi.reach(SCHEMA_USER, 'username'),
+
+    releaseId: Joi.number()
+        .integer()
+        .positive()
+        .optional()
+        .label('Release id'),
+
+    releaseName: Joi.string()
```
Comment (tk3fftk) [inline core/scm.js]: Does "release" include "tag"? If so, I think these field names are confusable.

Comment (tk3fftk) [reply core/scm.js]: I found it does not include "tag" in other pr.

File: core/scm.js [hunk 2]
```diff
@@ -187,7 +187,24 @@ const SCHEMA_HOOK = Joi.object().keys({
         .required()
         .label('Type of the event'),
 
-    username: Joi.reach(SCHEMA_USER, 'username')
+    username: Joi.reach(SCHEMA_USER, 'username'),
+
+    releaseId: Joi.number()
+        .integer()
```
Comment (wdstar) [inline core/scm.js]: We should use `string`. Maybe another SCM's release IDs include non-numbers.

Comment (kkokufud) [reply core/scm.js]: Thank you your comments. I changed type of releaseId number to string.

**Pull request merged**